test/test_job_archive_interface.py

```diff
@@ -47,63 +57,72 @@ def setUpClass(self):
             );"""
         )
 
-        # add sample jobs to job-archive database
-        id = 100
-        userid = 1234
-        username = "user" + str(userid)
-        t_submit = 1000
-        t_sched = 1005
-        t_run = 1010
-        t_cleanup = 1015
-        t_inactive = 1020
-        for i in range(4):
-            try:
-                jobs_conn.execute(
-                    """
-                    INSERT INTO jobs (
-                        id,
-                        userid,
-                        username,
-                        ranks,
-                        t_submit,
-                        t_sched,
-                        t_run,
-                        t_cleanup,
-                        t_inactive,
-                        eventlog,
-                        jobspec,
-                        R
+        def populate_job_archive_db(jobs_conn, userid, username, ranks, num_entries):
```

Comment:
Might be good to add `runtime`/`elapsed` as an argument to this function so that you can test with jobs that have different runtimes.